### PR TITLE
Use value on <select> instead of setting selected on <option>

### PR DESCRIPTION
**Pull request by clarkdo**

### CODE REVIEW

File: fixtures/attribute-behavior/src/App.js [hunk 1]
```diff
@@ -958,35 +958,25 @@ class App extends React.Component {
...
-            <option
-              selected={this.state.sortOrder === GROUPED_BY_ROW_PATTERN}
-              value={GROUPED_BY_ROW_PATTERN}>
+            <option value={GROUPED_BY_ROW_PATTERN}>
               grouped by row pattern :)
             </option>
           </select>
-          <select onChange={this.onUpdateFilter}>
-            <option selected={this.state.sortOrder === ALL} value={ALL}>
+          <select value={this.state.sortOrder} onChange={this.onUpdateFilter}>
```
Comment (nhunzaker) [inline fixtures/attribute-behavior/src/App.js]: I think this is wrong (not your doing). Could you change this to `this.state.filter`? It looks like this dropdown isn't working.

Comment (clarkdo) [reply fixtures/attribute-behavior/src/App.js]: @nhunzaker Done, thanks for the reviewing.

**Comment by clarkdo** [comment]
@nhunzaker Done, thanks for the reviewing.

**Comment by nhunzaker** [comment]
Awesome. This is great. Thank you for your contribution!

**Pull request merged**